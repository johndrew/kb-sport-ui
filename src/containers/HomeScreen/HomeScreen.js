import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import EventsIcon from '../../components/presentational/EventsIcon/EventsIcon';
import LiftersIcon from '../../components/presentational/LiftersIcon/LiftersIcon';
import './HomeScreen.scss';

export default class HomeScreen extends Component {
    render() {
        return (
            <div className="homeScreen__container">
                <Link to="/events"><EventsIcon /></Link>
                <Link to="/lifters"><LiftersIcon /></Link>
            </div>
        );
    }
}