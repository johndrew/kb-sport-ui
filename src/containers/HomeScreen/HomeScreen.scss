.homeScreen__container {
    display: grid;
    grid-template-columns: auto auto;
    width: 50%;
    margin: 0px auto;
}