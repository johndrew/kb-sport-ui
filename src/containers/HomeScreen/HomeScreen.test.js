import React from 'react';
import { shallow } from 'enzyme';
import HomeScreen from './HomeScreen';

describe(__filename, () => {

    it('renders correctly', () => {

        const wrapper = shallow(<HomeScreen />);

        expect(wrapper).toMatchSnapshot();
    });
});