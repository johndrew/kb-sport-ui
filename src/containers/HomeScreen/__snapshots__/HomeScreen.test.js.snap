// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Users/taylorjohndrew/Dev/projects/kb-sport-ui/src/containers/HomeScreen/HomeScreen.test.js renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomeScreen />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          replace={false}
          to="/events"
>
          <EventsIcon />
</Link>,
        <Link
          replace={false}
          to="/lifters"
>
          <LiftersIcon />
</Link>,
      ],
      "className": "homeScreen__container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <EventsIcon />,
          "replace": false,
          "to": "/events",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <LiftersIcon />,
          "replace": false,
          "to": "/lifters",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            replace={false}
            to="/events"
>
            <EventsIcon />
</Link>,
          <Link
            replace={false}
            to="/lifters"
>
            <LiftersIcon />
</Link>,
        ],
        "className": "homeScreen__container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <EventsIcon />,
            "replace": false,
            "to": "/events",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <LiftersIcon />,
            "replace": false,
            "to": "/lifters",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
